docker-images/bundle/metadata/annotations.yaml

```diff
@@ -0,0 +1,8 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: stable
+  operators.operatorframework.io.bundle.channels.v1: stable, strimzi-0.21.x
```

Comment:
As above we should probably have some logic for deciding the channel from the `release.version` file?

Reply:
Just updated the direction of the PR and have dropped support for generating the metadata files for now since we maintain these versions in OperatorHub